pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -93,7 +95,7 @@
   private static final long DEFAULT_SEGMENT_UPLOAD_TIMEOUT_IN_MILLIS = 600_000L; // 10 minutes
   private static final int DEFAULT_REALTIME_SEGMENT_METADATA_COMMIT_NUMLOCKS = 64;
   private static final boolean DEFAULT_ENABLE_STORAGE_QUOTA_CHECK = true;
-  private static final boolean DEFAULT_ENABLE_SEGMENT_LEVEL_VALIDATION = true;
+  private static final int DEFAULT_SEGMENT_LEVEL_VALIDATION_INTERVALS = 24;
```

Comment:
Should we keep this time bound instead of count bound? If the validation runs more frequently, then we'll reach the count sooner, right? We could track last segment validation time and define a segment-validation-interval-hours to control the interval?

Reply:
Done